### PR TITLE
release-notes: add release notes for the v93 release

### PR DESCRIPTION
**Pull request by ricardosalveti**
# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Release notes for the LmP v93 release.

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

File: source/release-notes/rn_v93.inc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+New Features
+------------
+
+ * Composectl is now used for managing compose apps (replacing skopeo, which was removed)
+
+Aktualizr-Lite Updates
+----------------------
+
+ * TODO
```
Comment (ricardosalveti) [inline source/release-notes/rn_v93.inc:34]: @mike-sul @detsch ^^

**Comment by doanac** [comment]
Docs for 7f23aff3c922efb25912795dff8913968ab003ee are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2636/docs/artifacts/html/index.html

File: source/release-notes/rn_v93.inc (hunk 1 above)
Comment (mike-sul) [reply source/release-notes/rn_v93.inc:34]:
```
Changes:

- Move to usage of the new `composectl` utility.
- Added offline update support to the public API.
- Support of the hybrid/mixing offline and online update.
- Added the CLI API - wrapper over the public API to streamline a custom CLI SOTA client.
```

Minor fixes & improvements in the offline update implementation and the calculation of the app update size.

**Review by kprosise — APPROVED** [review]
So far it lgtm

**Review by MrCry0 — APPROVED** [review]

File: source/release-notes/rn_v93.inc (hunk 1 above)
Comment (ricardosalveti) [reply source/release-notes/rn_v93.inc]: thanks!

**Comment by ricardosalveti** [comment]
@kprosise updated to include the ak-lite changes, can be merged if you are fine with it.

**Pull request merged**